src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -43,6 +43,7 @@ public function __construct()
             100 => [
                 new ResolveClassPass(),
                 new RegisterAutoconfigureAttributesPass(),
+                new RegisterAliasAttributesPass(),
```

Comment:
can't this work with $container->registerAttributeForAutoconfiguration() instead of a new pass?

Reply:
@nicolas-grekas Yes, it could, but if I'm not mistaking then this feature wouldn't work without the FrameworkBundle. This way all you need is the dependency-injection component.

Reply:
Is that really an issue?
There is no need to create extra work and maintenance if there is no users for it...

Reply:
@nicolas-grekas Nope, not really an issue, just an observation. I'll update the PR to use `registerAttributeForAutoconfiguration`.

Reply:
@nicolas-grekas Done, I had to pass some additional arguments to the closure in `AttributeAutoconfigurationPass`.